TicketsDemo.Data/Entities/TicketAggregate/Holiday.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TicketsDemo.Data.Entities
+{
+    public class Holiday
+    {
+        public string Id { get; set; }
```

Comment:
please add synthetic id of type Id.

With current approach you will be forced to generate ids manually for every record, which will be annoying (especially given that holidays with the same names will repeat over different years).

Holiday name should be added as separate field